tests/components/notify/test_facebook.py

```diff
@@ -0,0 +1,134 @@
+"""The test for the Facebook notify module."""
+import unittest
+import requests_mock
+
+from six.moves.urllib import parse as urlparse
```

Comment:
This can just be
```python
from urllib import parse
```
We're python3 exclusively.

Reply:
actually, i think i don't even need this lib at all. i'll kill it!